Dockerfile

```diff
@@ -12,6 +12,7 @@ ENV POSTGIS_VERSION 2.3.1+dfsg-1.pgdg80+1
 RUN apt-get update \
       && apt-get install -y --no-install-recommends \
            postgresql-$PG_MAJOR-postgis-$POSTGIS_MAJOR=$POSTGIS_VERSION \
+           postgresql-$PG_MAJOR-postgis-scripts \
```

Comment:
@dsteinich -- I'm fine with it. I don't mind addressing the merge conflicts in my PR.